src/pymor/vectorarrays/interfaces.py

```diff
@@ -838,12 +848,20 @@ def _create_random_values(shape, distribution, random_state, **kwargs):
         high = kwargs.get('high', 1.)
         if high <= low:
             raise ValueError
-        return random_state.uniform(low, high, shape)
+        if _is_complex_dtype(dtype):
+            # TODO the branches may not have a comparable EV
```

Comment:
What does this mean exactly?

Reply:
I guess EV is ambiguous? I mean expected value here. And that in turn was my way of saying "I do not know if that this is the right way to implement a comparable complex valued distribution".

Reply:
I'm pretty sure the random vector (X, Y) has a uniform distribution over a rectangle if and only if X and Y both have a uniform distribution (if that's what you're worried about).

Reply:
Alright. So normal dist is also OK then?

Reply:
Yes, at least for diagonal covariance matrix, which is probably fine for us.